lib/active_model/telephone_number_validator.rb

```diff
@@ -1,9 +1,23 @@
 class TelephoneNumberValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    country = options[:country].call(record) if options.key?(:country)
     valid_types = options.fetch(:types, [])
-    args = [value, country, valid_types]
+    args = [value, country(record), valid_types]
+    record.errors.add(attribute, message) if TelephoneNumber.invalid?(*args)
```

Comment:
Can we add a test for the `message` functionality to ensure that when the record is invalid, the correct message is added to the errors object?

Reply:
Oh yes, I kind of overlooked that. Of course I will. Thank you!

Reply:
should be good now!